priv/templates/_ginger_edit_content_status_label.event.tpl

```diff
@@ -0,0 +1,13 @@
+<div class="ginger-edit__aside-item ginger-edit__aside" id="ginger-edit__aside">
+    <h3 class="section-title">{_ Status Label _}</h3>
+    <label class="control-label" for="status-label">{_ Status labels are visible on contributions and meetups _}</label>
+    <div class="form-group row">
+        <select id="status-label" name="status_label" class="form-control c-status-select">
+            <option value="">{_ Select a status _}</option>
+            <option value="Preparation" {% if id.status_label == "Preparation" %}selected{% endif %}>{_ Preparation _}</option>
+            <option value="Meetup" {% if id.status_label == "Meetup" %}selected{% endif %}>{_ Meetup _}</option>
+            <option value="Insights" {% if id.status_label == "Insights" %}selected{% endif %}>{_ Insights _}</option>
+            <option value="Discussion" {% if id.status_label == "Discussion" %}selected{% endif %}>{_ Discussion _}</option>
```

Comment:
Instead of hardcoding the statuses here, is it not possible to find them?

I see that in `priv/templates/_ginger_edit_content_status_label.tpl` we do:
```
        {% with m.search[{query cat_exact=`status_keyword`}] as statuses %}
```

Reply:
Those are the ones for the KG (written in the schema) the hardcoded ones are the ones for the contributions. We could re-write the logic to make another category of keywords. Although i feel like since they will just be assigned as a property this was quicker

Reply:
oh, right, I see. Never mind then 👍